Sources_v3/StreamChatUI/ChatChannelList/ChatChannelListItemView.swift

```diff
@@ -21,6 +23,19 @@ open class ChatChannelListItemView<ExtraData: ExtraDataTypes>: View, UIConfigPro
     }
     
     // MARK: - Subviews
+
+    public private(set) lazy var actionButtonStack: UIStackView = {
+        let button = UIButton().withoutAutoresizingMaskConstraints
+        button.setImage(UIImage(named: "icn_delete", in: .streamChatUI), for: .normal)
+        button.backgroundColor = uiConfig.colorPalette.channelListActionsBackgroundColor
+        button.tintColor = uiConfig.colorPalette.channelListActionDeleteChannel
+        button.widthAnchor.constraint(equalTo: button.heightAnchor).isActive = true
+
+        let stack = UIStackView(arrangedSubviews: [button]).withoutAutoresizingMaskConstraints
+        stack.axis = .horizontal
+        stack.alignment = .fill
+        return stack
```

Comment:
Hm, cannot get this part 😄 Why we need to wrap the button into stack view? Why not to go with just a button?

Reply:
I guess you are right, I just was expecting to add more buttons, so I started with stackView instead 😅